config/prod/nf-vcf-workflow.yaml

```diff
@@ -0,0 +1,45 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-v6.yaml
```

Comment:
Nope. See PR here for ubuntu instance: https://github.com/Sage-Bionetworks/sandbox-provisioner/pull/121/files